docs/source/tutorial/tutorial_2.mdx

```diff
@@ -588,17 +594,15 @@ You'll see a warning in GraphiQL:
 
 ![variable dollar sign id of type ID was used in position expecting type ID exclamation point](images/graphiql_nullable_conflict.png)
 
-This bit can be VERY confusing for Swift developers in particular, because GraphQL's assumptions about nullability are essentially the opposite of Swift's. 
+This bit can be confusing, because GraphQL's assumptions about nullability are different from Swift's: 
```

Comment:
I think i'd like to keep `for Swift Developers in particular` here, because it's not an inherently confusing thing